src/bot-handlers/tests/disputer-test.js

```diff
@@ -0,0 +1,526 @@
+/* eslint-disable no-loop-func, max-len */
```

Comment:
Would it be helpful to add a comment that indicates most of this code came from UMA source file xxx?